source/workflowapi/external_resources.json

```diff
@@ -120,11 +120,24 @@
               "Version": "2012-10-17",
               "Statement": [
                 {
-                  "Effect": "Allow",
                   "Action": [
-                    "sqs:*"
+                      "s3:GetObject",
+                      "s3:DeleteObject"
                   ],
-                  "Resource": {"Ref": "SqsQueueArn"}
+                  "Resource": {"Fn::Sub": "arn:aws:s3:::${DataPlaneBucket}/*"},
+                  "Effect": "Allow"
+                },
+                {
+                    "Action": "s3:ListBucket",
+                    "Resource": {"Fn::Sub": "arn:aws:s3:::${DataPlaneBucket}"},
+                    "Effect": "Allow"
+                },
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                      "sqs:*"
```

Comment:
Can you scope this down?  Like this: https://github.com/awslabs/aws-media-insights-engine/blob/1c641a43894f31b987a9a0ab906d0a35753ecbeb/deployment/media-insights-stack.yaml#L255

Reply:
I didn't change the SQS policy for this PR except for formatting it.  I'll add a separate issue for that.